test/Repos.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+require('chai').use(require('sinon-chai'));
+require('dotenv').config();
```

Comment:
@ronanfitz I wonder if this is necessary here under the assumption it is probably invoked already in the source code.